src/ml_board/backend/websocket_api/websocket_server_dummy.py

```diff
@@ -12,7 +15,7 @@ def callback(self):
 
 class WebSocketServer:
```

Comment:
Do we still need this now that we have dumm_server.py?

Reply:
I personally don't use this anymore!